diff --git a/dist/api/openai-codex-responses.js b/dist/api/openai-codex-responses.js
index bb9fa794ee6cb9f73149dda5733b286941bbb550..0ee5a9fdac4f8aa83a0ee2c5681df71127ba4e7b 100644
--- a/dist/api/openai-codex-responses.js
+++ b/dist/api/openai-codex-responses.js
@@ -1242,15 +1242,15 @@ function buildBaseCodexHeaders(initHeaders, additionalHeaders, accountId, token)
         }
     }
     headers.set("Authorization", `Bearer ${token}`);
-    headers.set("chatgpt-account-id", accountId);
-    headers.set("originator", "pi");
-    const userAgent = _os ? `pi (${_os.platform()} ${_os.release()}; ${_os.arch()})` : "pi (browser)";
+    headers.set("ChatGPT-Account-ID", accountId);
+    headers.set("originator", "codex_cli_rs");
+    const userAgent = _os ? `codex_cli_rs/0.0.1 (${_os.platform()} ${_os.release()}; ${_os.arch()})` : "codex_cli_rs/0.0.1";
     headers.set("User-Agent", userAgent);
+    headers.set("version", "0.0.1");
     return headers;
 }
 function buildSSEHeaders(initHeaders, additionalHeaders, accountId, token, sessionId) {
     const headers = buildBaseCodexHeaders(initHeaders, additionalHeaders, accountId, token);
-    headers.set("OpenAI-Beta", "responses=experimental");
     headers.set("accept", "text/event-stream");
     headers.set("content-type", "application/json");
     if (sessionId) {
